platform-server-kernel/src/main/java/uno/cod/platform/server/core/repository/ChallengeRepository.java

```diff
@@ -24,9 +24,8 @@
     Challenge findOneWithEndpointAndTasks(@Param("id") Long id);
 
     @Query("SELECT challenge FROM Challenge challenge " +
-            "LEFT JOIN FETCH challenge.tasks " +
```

Comment:
function is named `findAllWithTasks` and you don`t fetch the tasks anymore. If you get multiple results for challenge you can `SELECT DISTINCT' instead of 'SELECT'